server/api/screenshot/screenshotController.js

```diff
@@ -15,33 +15,35 @@ exports.list = function(req, res, next){
   });
 };
 
+/**
+ * create
+ * ======
+ * Takes screenshot of specified URL, creates new Screenshot in DB, and
+ * adds screenshot to the user images.
+ */
 exports.create = function(req, res, next) {
-  var url = req.body.url;
   var username = req.params.username;
-  var annotatedImage = req.body.annotatedImage;
```

Comment:
Right now I'm putting the imageUrl from the takeScreenshot into both `originalImage` and `annotatedImage`. It'll be something like a placeholder until we add that feature. Seem reasonable?